### PR TITLE
convert-HA-xsight-proposal

### PR DESCRIPTION
**Pull request by mmiele**
Converted email content to markdown file

### CODE REVIEW

**Comment by mmiele** [comment]
This content might be related to [xsight-labs-ha-proposal-v1.md](xsight-labs-ha-proposal-v1.md) please verify and advice.

File: documentation/high-avail/design/xsight-labs-ha-proposal-new-ideas.md [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+example, there may be less value in buffering/holding syn packets. If the peer
+does not learn of the syn and there is a failover. The syn-ack will be dropped
+by the peer and the syn will be retransmitted by the endpoint. Anyone may
+contribute a full definition and analysis any of the sender approaches above, or
+other possible approaches/optimizations, as an optional "standardized" DASH HA
+sender modes. These modes only affect the sender behavior. All sender modes use
+the same protocol definition and the receiver behavior will always remain
+simple, stateless and interoperable with all sender modes.
+
+![ha-state-sync-packet-format](images/ha-state-sync-packet-format.png)
```
Comment (chrispsommers) [inline documentation/high-avail/design/xsight-labs-ha-proposal-new-ideas.md:121]: .svg please

File: documentation/high-avail/design/xsight-labs-ha-proposal-new-ideas.md [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+may be truncated. The reply is simply the original HA state synchronization
+packet with a reply flag set and is possibly truncated (it is allowed, but not
+bandwidth optimized, for the receiver to not truncate the reply when requested).
+The sender may optionally include opaque information with each individual
+message in the synchronization packet and/or for the synchronization packet at a
+whole. When the reply is returned to the sender, the opaque information can be
+used in an implementation specific manner to accomplish stateless or stateful
+synchronization operations.
+
+![ha-sync-operations](images/ha-sync-operations.png)
```
Comment (chrispsommers) [inline documentation/high-avail/design/xsight-labs-ha-proposal-new-ideas.md:82]: .svg please

**Comment by chrispsommers** [comment]
The e-mail authors might have editable .svg files already, that image format would be preferable. Thanks.

**Review by budgrise — APPROVED** [review]
Looks good.
The xsight-labs-ha-proposal-v1.md doc proposed a different HA message format.
That was before we understood the requirement for supporting asymmetrical flows.
So this doc supersedes much (but not all) of the content of that earlier proposal.

File: documentation/high-avail/design/xsight-labs-ha-proposal-new-ideas.md (hunk 1 above)
Comment (mmiele) [reply documentation/high-avail/design/xsight-labs-ha-proposal-new-ideas.md]: done

File: documentation/high-avail/design/xsight-labs-ha-proposal-new-ideas.md (hunk 2 above)
Comment (mmiele) [reply documentation/high-avail/design/xsight-labs-ha-proposal-new-ideas.md]: done

**Comment by mmiele** [comment]
@budgrise For now we keep both docs: 
1. xsight-labs-ha-proposal-v1.md
2. xsight-labs-ha-proposal-new-ideas.md
Let me know if we need to rework/merge/rename them.

**Review by KrisNey-MSFT — APPROVED** [review]
Looks good to me, thanks.

**Comment by mmiele** [comment]
> The e-mail authors might have editable .svg files already, that image format would be preferable. Thanks.

Got them and used them.

**Pull request merged**